src/SelectMulti.vue

```diff
@@ -215,17 +215,21 @@
 		</label>
 		<ul :id="`${htmlId}-selected`" class="selected-options" :class="{ 'below-input': displayPillsBelowInput }">
 			<span :id="`${htmlId}-selected-option-pills`" style="display: none;">remove</span>
-			<li v-for="(option, index) in selectedOptions" :key="option.value">
-				<button
-					class="selected-option-pill"
-					:disabled="disabled"
-					type="button"
-					:aria-describedby="`${htmlId}-selected-option-pills`"
-					@click="removeOption(index)"
-				>
-					{{ option.label }}
-				</button>
-			</li>
+			<template v-for="(option, index) in selectedOptions" >
+				<li v-if="option.value" :key="option.value" >
+					<button
```

Comment:
I didn't find a better solution to fix issue when we use custom slot and havn't any selected options.
What I'm talking about:
![image](https://user-images.githubusercontent.com/44836424/96129288-1d8a8a00-0eff-11eb-855b-2763dbf4af3a.png)